README.md

```diff
@@ -25,3 +25,13 @@ cd /build
 ./test/derivative/derivative_test
 ./test/matrix-multiply/matrix_multiply_test
 ```
+
+On Casper, to run GPU tests:
+```
+module load cmake/3.22.0 gnu/12.1.0 cuda/11.4.0 nvhpc/22.2
```

Comment:
By loading `nvhpc/22.2`, I think the `gnu/12.1.0` module will be unloaded automatically.